src/main/java/me/jackint0sh/timedfly/database/MySQL.java

```diff
@@ -24,12 +23,33 @@ public AsyncDatabase connect(String host, int port, String database, String user
 
         try {
             conn = DriverManager.getConnection("jdbc:mysql://" + host + ":" + port + "/" + database
-                    + "?autoReconnect=true&useSSL=false", user, password);
+                    + "?autoReconnect=true&useSSL=" + useSSL, user, password);
             callback.handle(null, null);
         } catch (SQLException e) {
             callback.handle(e, null);
         }
         return this;
     }
 
+    // Legacy support or for any reason useSSL is not sent
```

Comment:
This can just call the other connect method but just provide false at the end.
Something like this:
```java
public foo(T bar) {
  // logic
}

public foo() {
  foo(bar)
}
```

But we can just delete this because I don't think someone else but this plugin uses it. If it was an API then yeah, we would need this.

Reply:
Agreed.  I added the secondary method to satisfy the requirement to implement due to the abstraction of AsyncDatabase via SQL extension.  It is either this or make MySQL class abstract.  What would be the best course of action? 

In the meantime, I'll add as follows:
`public AsyncDatabase connect(String host, int port, String database, String user, String password, Callback<String> callback) {
  	return connect(host, port, database, user, password, false, callback);
 }
`

Reply:
Ah yes, I forgot about the abstraction 😅
Lets just do the first suggestion to minimize it and not repeat the code

Reply:
Done.